src/locale-data/ko-KR.json

```diff
@@ -104,5 +104,17 @@
 			"percent": "%"
 		},
 		"groupSize": 3
+	},
+	"fileSize": {
+		"patterns": {
+			"fileSizePattern": "{number} {unit}"
```

Comment:
According [to the old terms](http://search.dev.d2l/source/xref/LSOne/lp/_lang/LangTerms/translations/ko-kr/Standard.xml#585), I think the pattern should not have a space: `{number}{unit}`